cli/src/command/config_transaction_create.rs

```diff
@@ -0,0 +1,304 @@
+        println!("Your Public Key:       {}", transaction_creator);
+        println!();
+        println!("⚙️ Config Parameters");
+        println!("Multisig Key:       {}", multisig_pubkey);
+        println!("Transaction Index:       {}", transaction_index);
+        println!("Vote Type:       {}", transaction_index);
+        println!();
+
+        let proceed = Confirm::new()
+            .with_prompt("Do you want to proceed?")
+            .default(false)
+            .interact()?;
+        if !proceed {
+            println!("OK, aborting.");
+            return Ok(());
+        }
+        println!();
+
+        let progress = ProgressBar::new_spinner().with_message("Sending transaction...");
+        progress.enable_steady_tick(Duration::from_millis(100));
+
+        let blockhash = rpc_client
+            .get_latest_blockhash()
+            .await
+            .expect("Failed to get blockhash");
+
+        let message = Message::try_compile(
+            &transaction_creator,
+            &[
+                Instruction {
+                    accounts: ConfigTransactionCreateAccounts {
+                        creator: transaction_creator,
+                        multisig,
+                        rent_payer: transaction_creator,
+                        transaction: transaction_pda.0,
+                        system_program: solana_sdk::system_program::id(),
+                    }
+                    .to_account_metas(Some(false)),
+                    data: ConfigTransactionCreateData {
+                        args: ConfigTransactionCreateArgs {
+                            actions: vec![config_action.unwrap()],
+                            memo,
+                        },
+                    }
+                    .data(),
+                    program_id,
+                },
+                Instruction {
+                    accounts: ProposalCreateAccounts {
+                        creator: transaction_creator,
+                        multisig,
+                        rent_payer: transaction_creator,
+                        proposal: proposal_pda.0,
+                        system_program: solana_sdk::system_program::id(),
+                    }
+                    .to_account_metas(Some(false)),
+                    data: ProposalCreateData {
+                        args: ProposalCreateArgs {
+                            draft: false,
+                            transaction_index,
+                        },
+                    }
+                    .data(),
+                    program_id,
+                },
+            ],
+            &[],
+            blockhash,
+        )
+        .unwrap();
+
+        let transaction = VersionedTransaction::try_new(
+            VersionedMessage::V0(message),
+            &[&*transaction_creator_keypair],
+        )
+        .expect("Failed to create transaction");
+
+        match rpc_client.send_and_confirm_transaction(&transaction).await {
```

Comment:
The whole block of creating and sending txs (everything but individual ixs) scan probably be extracted into a utility function to reduce duplication. Don't necessarily need to do it now

Reply:
added